### ISSUE TITLE
Support events with '.'

### ISSUE THREAD

**Issue by MichaelRFairhurst** [dart-lang/angular_analyzer_plugin#423]
I actually don't know what `(x.y)` means, but we need to support this.

Autocompletion can come later, validation is the important part. Though if autocomplete is simple....

**Comment by zoechi** [comment]
It might not mean anything, just that the event name contains a `.` like
```dart
@Input('my.event') myEvent;
```
and perhaps
```
someElement.dispatchEvent(new CustomEvent('my.event'));`
```
could also be some event fired from JavaScript code.

**Comment by MichaelRFairhurst** [comment]
Fixed in #430, though we don't yet support custom events. That should probably be treated much like #454 will open a new bug for that
